src/java.compiler/share/classes/javax/lang/model/element/package-info.java

```diff
@@ -36,7 +36,7 @@
  * appearing inside method bodies, such as local variables and
  * anonymous classes.
  *
- * <p>When used in the context of annotation processing, an accurate
+ * <p><a id="accurate_model">When used in the context of annotation processing, an accurate
```

Comment:
It would be more correct to just put the id on the `<p>` tag and drop the `<a>`.

Reply:
Fixed.